docs/intentReaderHandlers/LifecycleRecordHandler.md

```diff
@@ -0,0 +1,205 @@
+# LifecycleRecordHandler
+
+## Overview
+
+LifecycleRecordHandler is invoked when the task specified in the intent has the value "startLifecycleRecording/stopLifecycleRecording". When the intent received has the task "startLifecycleRecording", FCA starts listening for lifecycle events and will stop listening once it gets the intent with the task "StopLifecycleRecording", and the event list will be saved.
+
+**`StartLifecycleRecording`** - This handler intent is identified when the task field has value as "startLifecycleRecording". This handler is used to start listening lifecycle events.
+
```

Comment:
This handler intent is identified when the task field has value as "startLifecycleRecording". This handler is used to start listening lifecycle events. 
to be changed to 
**_This handler intent is identified when the task field has value as "startLifecycleRecording". This handler is used to start listening to lifecycle events._**

Reply:
updated